src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -2,10 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Left off review here. Will pick back up again tomorrow. @gafter, for future annotation PRs I would request that you keep them smaller, please. Every file that is annotated has to be gone through in its entirety, so a 1000-line diff means several thousands lines of code to review.